### PR TITLE
New View Tests

### PR DESCRIPTION
**Pull request by Awinja-j**
#### This PR Introduces tests files for the available views which are:

- Doctors
- Clinical officers
- Nurses
- Health Facilities
- NHIF Inpatient
- NHIF Outpatient
- NHIF Outpatient Civil Servants

#### This tests can be run by using the following commands:

`nosetests healthtools/tests`
`nosetests healthtools/tests/doctor.py`
`nosetests healthtools/tests/nurse.py`
`nosetests healthtools/tests/clinical_officer.py`
`nosetests healthtools/tests/health_facilities.py`
`nosetests healthtools/tests/nhif_inpatient.py`
`nosetests healthtools/tests/nhif_outpatient.py`
`nosetests healthtools/tests/nhif_outpatient_cs.py`

#### Relevant Github Issue:

https://github.com/CodeForAfricaLabs/HealthTools.API/issues/46

### CODE REVIEW

File: healthtools/manage.py [hunk 1]
```diff
@@ -1,3 +1,10 @@
+import os
+import inspect
+import sys
+currentdir = os.path.dirname(os.path.abspath(
+    inspect.getfile(inspect.currentframe())))
+parentdir = os.path.dirname(currentdir)
+sys.path.insert(0, parentdir)
```
Comment (DavidLemayian) [inline healthtools/manage.py]: What are these four lines of code for?

**Comment by DavidLemayian** [comment]
This is the exact same PR I closed a couple of days ago. Did you open this one by mistake?  https://github.com/CodeForAfricaLabs/HealthTools.API/pull/59#issuecomment-333814243

**Comment by Awinja-j** [comment]
@DavidLemayian This is a new repo. i added new test cases that i feel make use of the iterators in the documents.

File: healthtools/manage.py (hunk 1 above)
Comment (DavidLemayian) [reply healthtools/manage.py]: @Awinja-Andela ☝️

**Comment by DavidLemayian** [comment]
Hi @Awinja-Andela. Going to re-open this as there are many ways to skin a cat.

**Comment by Awinja-j** [comment]
@DavidLemayian please check on changes i have made.

**Review by DavidLemayian — APPROVED** [review]

**Pull request merged**